drivers/SmartThings/matter-lock/fingerprints.yml

```diff
@@ -15,7 +15,34 @@ matterManufacturer:
     deviceLabel: Aqara Smart Lock U200
     vendorId: 0x115F
     productId: 0x2802
-    deviceProfileName: lock-lockalarm-nobattery
+    deviceProfileName: lock-user-pin
+  - id: "4447/10241"
+    deviceLabel: Aqara Smart Lock U300
+    vendorId: 0x115F
+    productId: 0x2801
+    deviceProfileName: lock-user-pin
+#eufy
+  - id: 5427/1
```

Comment:
`id` values should be surrounded with "" to prevent them from being evaluated as mathematical expressions